### PR TITLE
Fix typo in elastic agent dashboard

### PR DESCRIPTION
**Pull request by smith**
## What does this PR do?

Fix typo in elastic agent dashboard

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.

### CODE REVIEW

**Review by kaiyan-sheng — APPROVED** [review]

File: packages/elastic_agent/kibana/dashboard/elastic_agent-f47f18cc-9c7d-4278-b2ea-a6dee816d395.json [hunk 1]
```diff
@@ -90,4 +90,4 @@
         }
     ],
     "type": "dashboard"
-}
\ No newline at end of file
+}
```
Comment (ycombinator) [inline packages/elastic_agent/kibana/dashboard/elastic_agent-f47f18cc-9c7d-4278-b2ea-a6dee816d395.json:93]: I suspect this newline change is why the CI build is failing. CI is pretty strict about formatting.

From anywhere under the `packages/elastic_agent` folder, run `elastic-package format`. That should fix up any formatting.

Comment (smith) [reply packages/elastic_agent/kibana/dashboard/elastic_agent-f47f18cc-9c7d-4278-b2ea-a6dee816d395.json:93]: I made this change through the GitHub UI. I don't have this project checked out yet.

Comment (ycombinator) [reply packages/elastic_agent/kibana/dashboard/elastic_agent-f47f18cc-9c7d-4278-b2ea-a6dee816d395.json:93]: Ah, I see. Let me push up a commit that fixes the formatting.

**Review by mtojek — CHANGES_REQUESTED** [review]
Could you please bump up the version and update changelog?

**Comment by smith** [comment]
[Fix typo in elastic agent dashboard #962](https://trello.com/c/PyljzHVn/3-fix-typo-in-elastic-agent-dashboard-962)

**Review by ycombinator — APPROVED** [review]
LGTM.

**Review by mtojek — APPROVED** [review]

**Pull request merged**